ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/AlbumListFragment.kt

```diff
@@ -0,0 +1,110 @@
+package org.moire.ultrasonic.fragment
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.LiveData
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.RecyclerView
+import org.koin.core.component.KoinApiExtension
+import org.moire.ultrasonic.R
+import org.moire.ultrasonic.domain.MusicDirectory
+import org.moire.ultrasonic.domain.MusicFolder
+import org.moire.ultrasonic.util.Constants
+
+/**
+ * Displays a list of Albums from the media library
+ * TODO: Check refresh is working
+ */
+@KoinApiExtension
+class AlbumListFragment : GenericListFragment<MusicDirectory.Entry, AlbumRowAdapter>() {
+
+    /**
+     * The ViewModel to use to get the data
+     */
+    override val listModel: AlbumListModel by viewModels()
+
+    /**
+     * The id of the main layout
+     */
+    override val mainLayout: Int = R.layout.generic_list
+
+    /**
+     * The id of the refresh view
+     */
+    override val refreshListId: Int = R.id.generic_list_refresh
+
+    /**
+     * The id of the RecyclerView
+     */
+    override val recyclerViewId = R.id.generic_list_recycler
+
+    /**
+     * The id of the target in the navigation graph where we should go,
+     * after the user has clicked on an item
+     */
+    override val itemClickTarget: Int = R.id.trackCollectionFragment
+
+    /**
+     * Whether to show the folder selector
+     */
+    override var folderHeaderEnabled = false
```

Comment:
I think this shouldn't be always false here, at least now the folder selector for the albums doesn't work at all...